src/connectors/stats/__tests__/connectStats-test.js

```diff
@@ -120,7 +120,17 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/stats/js/#c
       stats.init(createInitOptions({ helper, state: helper.state }));
 
       const results = new SearchResults(helper.state, [
-        createSingleSearchResponse(),
+        createSingleSearchResponse({
+          hits: [
+            { brand: 'samsung', objectID: '1' },
+            { brand: 'apple', objectID: '2' },
+            { brand: 'sony', objectID: '3' },
+            { brand: 'benq', objectID: '4' },
+            { brand: 'dyson', objectID: '5' },
+          ],
+          hitsPerPage: 3,
+          query: 'apple',
+        }),
```

Comment:
@Haroenv I've updated the `singleSearchResponse` so it's different than the `getRenderState` one, not sure if that's what you meant.

Reply:
what I meant is that these two asserts should be in their own `it/test` function, since they don't test the same case (init/render). Sorry for being unclear!

Reply:
Oh I got it now! Thanks, I'll update that :D

Reply:
Let me know if that's what you had in mind https://github.com/algolia/instantsearch.js/pull/4565/commits/f4b71de2481934f52838d6b7434f55b384175adf